docs/source/contents/5_technical_metadata_properties.rst

```diff
@@ -0,0 +1,245 @@
+####################
+Technical Properties
+####################
+
+*****
+About
+*****
+
+While our MODS to RDF mapping aims to address how descriptive properties should be mapped in our next system, this
+section aims to explain how to map existing technical metadata properties our metadata properties that we need going
+forward.  When a technical metadata property may exist, its current mapping is described.
```

Comment:
I'm not sure what you mean by "When a technical metadata property may exist, its current mapping is described." The "may exist" portion is confusing to me. If there's a current mapping and it exists, why say "may"?

Reply:
👍🏻